modules/monitoring/library/Monitoring/Web/Navigation/Renderer/BackendAvailabilityNavigationItemRenderer.php

```diff
@@ -59,6 +64,16 @@ public function getCount()
             }
             $count = 0;
             $titles = array();
+            if (! (bool) $programStatus->active_host_checks_enabled) {
+                $count = 1;
```

Comment:
I know, you just did what the previous implementation also did. But don't you think increasing `$count` with each case is a reasonable change as well? Multiple titles are already shown, though the count is always `1` :thinking:

Reply:
Good call - I've changed this now.